atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -176,7 +177,16 @@ private long waitForSchemaMutationLock() {
 
                 List<Column> expected = ImmutableList.of(lockColumnWithValue(GLOBAL_DDL_LOCK_CLEARED_VALUE));
 
-                CASResult casResult = writeDdlLockWithCas(client, expected, ourUpdate);
+                CASResult casResult;
+                try {
+                    casResult = writeDdlLockWithCas(client, expected, ourUpdate);
```

Comment:
To avoid splitting declaration and assignment, I'd like to see the `try/catch` block extracted into a method - perhaps called `tryWritingDdlLockWithCas(client, expected, ourUpdate)`.

Reply:
Or perhaps `writeDdlLockWithCasCleaningUpAfterFailure` - the long method name isn't a problem, as its scope is small.